docs/operator_parameters.md

```diff
@@ -0,0 +1,299 @@
+Postgres Operator Configuration
+===============================
+
+Postgres operator is configured via a ConfigMap defined by the
+*CONFIG_MAP_NAME* environment variable. Variable names are underscore-separated
+words.
+
+Available Configuration Parameters
+----------------------------------
+
+### General
+* **etcd_host**
+  Etcd *host:port* for Patroni. Not required when Patroni native Kubernetes
+  support is used. The default is empty (use Kubernetes-native DCS).
+
+* **docker_image**
+  Spilo docker image for Postgres instances. For production, don't rely on
+  defaults, as it might be too old. Instead, build your own Spilo image from
+  the [github repository](github.com/zalando/spilo).
+
+* **workers**
+  number of working routines the operator spawns to process requests to
+  create/update/delete/sync clusters concurrently. The default is *4*.
+
+* **max_instances**
+  operator will cap the number of instances in any managed Postgres cluster up
+  to the value of this parameter. When *-1* is specified, no limits are applied.
+  The default is *-1*.
+
+* **min_instances**
+  operator will run at least the number of instances for any given Postgres
+  cluster equal to the value of this parameter. When *-1* is specified, no limits
+  are applied. The default is *-1*.
+
+* **resync_period**
+  period between consecutive sync requests. The default is '*5m*'. 
+
+### Postgres users
+* **super_username**
+  postgres superuser name to be created by *initdb*. The default is
```

Comment:
Is there any particular pattern why some are 'Postgres' with capital P, some are 'postgres' without capital, and some are PostgreSQL (in other part of documentation)

Reply:
Good point, lowercased those that are not leading sentences.